samples/test-samples.cmake

```diff
@@ -40,7 +40,8 @@ endif ()
 
 # Set SAMPLES_LIST so that helloworld becomes the first if BUILD_ENCLAVES=ON.
 if (BUILD_ENCLAVES)
-  set(SAMPLES_LIST helloworld file-encryptor switchless host_verify)
+  set(SAMPLES_LIST file-encryptor helloworld host_verify log_callback
```

Comment:
Can you add log_callback to the samples readme?

Reply:
https://github.com/openenclave/openenclave/blob/master/samples/README_Linux.md and README_Windows.md

Reply:
Thanks, now it is updated.